nc/source/graphics/Graphics2.h

```diff
@@ -66,6 +65,7 @@ namespace nc::graphics
             std::unique_ptr<vulkan::Swapchain> m_swapchain;
             std::unique_ptr<vulkan::Commands> m_commands;
             vulkan::Renderer* m_renderer;
+            std::unique_ptr<vulkan::ResourceManager> m_resourceManager;
```

Comment:
This doesn't have to be a unique_ptr; we just did that because ResourceManager has mostly (all?) inline functions in its header. Will you put a todo here and in ResourceManager to fix these things?